pkg/trivyk8s/trivyk8s.go

```diff
@@ -101,11 +101,17 @@ func (c *client) GetArtifact(ctx context.Context, kind, name string) (*artifacts
 }
 
 func (c *client) getDynamicClient(gvr schema.GroupVersionResource) dynamic.ResourceInterface {
-	k8s := c.cluster.GetDynamicClient()
+	dclient := c.cluster.GetDynamicClient()
+
+	// ignore namespace if it is a cluster level resource
+	if k8s.IsClusterResource(gvr) {
```

Comment:
Maybe it will be less duplicate code if we 
```suggestion
	if k8s.IsClusterResource(gvr) || len(c.namespace) == 0 {
```
and remove L112,L113,115